src/pybreaker.py

```diff
@@ -587,14 +588,26 @@ def opened_at(self, now):
         """
         try:
             key = self._namespace('opened_at')
-            def set_if_greater(pipe):
-                current_value = pipe.get(key)
+
+            if self._cluster_mode:
+
+                current_value = self._redis.get(key)
                 next_value = int(calendar.timegm(now.timetuple()))
-                pipe.multi()
+
                 if not current_value or next_value > int(current_value):
-                    pipe.set(key, next_value)
+                    self._redis.set(key, next_value)
```

Comment:
That's it! Now the only thing I miss is a test case to exercise this code path here, to make sure it works as expected.